hummingbot/connector/gateway/gateway_price_shim.py

```diff
@@ -38,17 +38,17 @@ class GatewayPriceDeltaEntry:
 
 class GatewayPriceShim:
     """
-    Developer / QA tool for modifying the apparent prices on DEX connectors (e.g. uniswap) during integration tests.
+    Developer / QA tool for modifying the apparent prices on DEX connectors (e.g. amm) during integration tests.
```

Comment:
We have connector / exchange named uniswap
https://hummingbot.org/exchanges/uniswap/

this changes here not needed